docs/fundamentals/jcloud/autoscale.md

```diff
@@ -0,0 +1,54 @@
+# Autoscaling
+
+JCloud is Kubernetes based and one of the biggest benefits of Kubernetes is the ability to autoscale your applications
+on-demand. In JCloud, autoscaling function is naturally offered as well, so you can maintain the [serverless](https://en.wikipedia.org/wiki/Serverless_computing) deployments in a cost-effective way with no headache of setting the [right number of replicas](https://docs.jina.ai/how-to/scale-out/#scale-out-your-executor) anymore!
```

Comment:
I suggest we first tell the user about what JCloud can do, and then how it is done, instead of the other way around
```suggestion
In JCloud, demand-based autoscaling functionality is naturally offered thanks to the underlying Kubernetes architecture. This means that you can maintain [serverless](https://en.wikipedia.org/wiki/Serverless_computing) deployments in a cost-effective way with no headache of setting the [right number of replicas](https://docs.jina.ai/how-to/scale-out/#scale-out-your-executor) anymore!
```

Reply:
I like this change!